modules/exploits/linux/http/pulse_secure_file_disclosure.rb

```diff
@@ -0,0 +1,73 @@
+# Quick hackish exploit for the CVE-2019-11510
+# made with help from Alyssa Herrera
+# Sorry for using your exploit as a template Carlos E. Vieira
+# Grabs /etc/passwd as a proof of concept
+
+require 'msf/core'
+class MetasploitModule < Msf::Auxiliary
+	include Msf::Exploit::Remote::HttpClient
+	include Msf::Post::File
+	def initialize(info = {})
+		super(update_info(info,
+			'Name'           => 'Pulse Secure - System file leak',
+			'Description'    => %q{
+				Pulse Secure SSL VPN file disclosure via specially crafted HTTP resource requests.
+        This exploit reads /etc/passwd and displays it to the terminal.
+        This vulnerability affect ( 8.1R15.1, 8.2 before 8.2R12.1, 8.3 before 8.3R7.1, and 9.0 before 9.0R3.4
+			},
+			'References'     =>
+			    [
+			        [ 'URL', 'http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-11510' ]
+			    ],
+			'Author'         => [ '0xDezzy (Justin Wagner), Alyssa Herrera' ],
```

Comment:
Twitter handles are usually not permitted in the `Author` key. Instead, they're placed in comments following the Array entry. Additionally, the comments usually include some form of attribution. For example, something like this:

```suggestion
			'Author'         => [
			  'Alyssa Herrera', # Discovery and exploit
			  'Justin Wagner',  # 0xDezzy - Metasploit
			],
```

Note: I have no idea if the above is accurate, I haven't read the advisory. If Alyssa didn't provide an exploit, then use `Discovery and PoC`. If no PoC was provided, then simply use `Discovery`.